<launch>
    <!-- <node name="JetbotCamera" pkg="mas514" type="JetbotCamera.py" output="screen"/>
    <node name="ImageSaver" pkg="mas514" type="ImageSaver.py" output="screen"/> -->
    <node name="JetbotController" pkg="mas514" type="JetbotController.py" output="screen"/>
    <node name="MainController" pkg="mas514" type="MainController.py" output="screen"/> 
    <node name="WebController" pkg="mas514" type="WebController.py" output="screen"/>
    <node name="node_web_video_server" pkg="web_video_server" type="web_video_server"/>  
    <node name="Odometry" pkg="mas514" type="Odometry.py" output="screen"/>
    <node name="serial_node" pkg="rosserial_python" type="serial_node.py" output="screen">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="baud" value="115200"/>
  </node> 
    <!-- <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" >
        <arg name="port" value="9090"/>
    </include> -->
  <node name="InverseKinematics" pkg="mas514" type="InverseKinematics.py" output="screen"/> 

  <!-- <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 map odom 30" /> -->

</launch>
